js/src/pages/edit-paid-ads-campaign/index.js

```diff
@@ -168,7 +168,7 @@ const EditPaidAdsCampaign = () => {
 		const hasChange =
 			allValues.amount !== campaign.amount ||
 			! isEqual(
-				assetEntityGroup.display_url_path,
+				assetEntityGroup?.display_url_path,
 				allValues.display_url_path
 			) ||
```

Comment:
In the case where the `assetEntityGroup` is undefined, this condition will cause the `hasChange` value to always be true before changing any campaign assets on UI.

![image](https://github.com/user-attachments/assets/dedb3f21-c0c9-4052-8e48-d8b1f6ff24a4)

After this condition can be correctly determined, the next line also triggers the same error as #2791.

https://github.com/woocommerce/google-listings-and-ads/blob/e95ac381d507143540db2798eac404c26a613ac4/js/src/pages/edit-paid-ads-campaign/index.js#L174

Reply:
Thanks for catching this bug, I should've done more tests on it. I've added one commit to address it and it's ready for another round.